lib/utils/request.js

```diff
@@ -193,6 +192,72 @@ function MULTIGET (fhurls, where, cb) {
   });
 }
 
+/**
+ * Utility function to download files to a file path from a url
+ * @param params
+ * @param cb
+ */
+function downloadFile(params, cb){
+  var url = params.url;
+  var filePath = params.output;
+  var fileStream = fs.createWriteStream(filePath);
```

Comment:
Should fileStream also have an error listner? What happens if the specified filePath doesn't exist, or permissions are lacking?